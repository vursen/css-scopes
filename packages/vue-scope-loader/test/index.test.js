const compiler = require('./compiler');

test('zero config css scope', async () => {
  const stats = await compiler('./fixtures/bundle.js');
  const output = stats.toJson().modules[0].source

  console.log(stats.toJson())

  expect(output).toMatchSnapshot()
});

test('zero config vue scope', async () => {
  const stats = await compiler('./fixtures/bundle.js');
  const output = stats.toJson().modules[2].source

  expect(output).toMatchSnapshot()
});

// test('custom default scope path', async () => {
//   const stats = await compiler('./fixtures/bundle.js', {
//     defaultScopePath: './[name].html'
//   });

//   const output = stats.toJson().modules[0].source

//   expect(output).toMatchSnapshot()
// });
