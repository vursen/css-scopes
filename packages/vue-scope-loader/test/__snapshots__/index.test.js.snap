// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zero config css scope 1`] = `
"exports = module.exports = require(\\"../../../node_modules/css-loader/dist/runtime/api.js\\")(false);
// Module
exports.push([module.id, \\"\\\\n[__fixtures-components-Component-vue].host {\\\\n  font-size: 15px;\\\\n}\\\\n[__fixtures-components-Component-vue].host [__fixtures-components-Component2-vue].host {\\\\n  font-size: 15px;\\\\n}\\\\n\\", \\"\\"]);
"
`;

exports[`zero config vue scope 1`] = `
"var render = function() {
  var _vm = this
  var _h = _vm.$createElement
  var _c = _vm._self._c || _h
  return _vm._m(0)
}
var staticRenderFns = [
  function() {
    var _vm = this
    var _h = _vm.$createElement
    var _c = _vm._self._c || _h
    return _c(\\"div\\", { staticClass: \\"host\\" }, [
      _c(\\"div\\", { staticClass: \\"title\\" }, [_vm._v(\\"Title\\")])
    ])
  }
]
render._withStripped = true

export { render, staticRenderFns }"
`;
